services/movie.py

```diff
@@ -0,0 +1,41 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] | None = None,
+    actors_ids: list[int] | None = None,
+) -> QuerySet:
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids, actors__id__in=actors_ids
+        )
+
+    if genres_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        return Movie.objects.filter(actors__id__in=actors_ids)
+
+    return Movie.objects.all()

```

Comment:
Add only one return statement in the code. Remember that QuerSets are lazy and you can add `filter`  statements in case corresponding arguments are present